spec/Cabin/SingleWindow.vspec

```diff
@@ -16,15 +16,15 @@ IsOpen:
 
 Position:
   datatype: uint8
-  type: sensor
+  type: actuator
   min: 0
   max: 100
   unit: percent
   description: Window position. 0 = Fully closed 100 = Fully opened.
 
 IsChildLockEngaged:
```

Comment:
This could possibly be a good opportunity to discuss if this signal needs more improvements in the long run (not necessarily part of this PR). Is this actually a signal that is needed for every window or would a joint "cabin" signal be sufficient. In most cars it is a single button, and it disables all buttons except the buttons controlled by the driver. Are there vehicles were you can engage child lock individually for windows? If not, would it possibly suffice with a single signal.

One can also argue that our model does not consider that there typically are two switches for most windows (except for the driver window). If `IsChildLockEngaged==True` then the switch next to the window shall be disabled, but normally not the switch next to the driver. But with the current setup with a single switch below it is not possible by the VSS signals to identify which switch that is used.

Reply:
I am open to moving it out, either here or in a separate PR. I think `instantiate=false` would do the trick?

Reply:
@erikbosch 
Usually it is always one switch for Row2 windows. Might be logical to have this on higher level.